src/content/blog/hypervisor-virtio-mmio-registers.md

```
@@ -0,0 +1,359 @@
+
+- ディスクイメージの読み書き
+- `VirtioBlkReq` 構造体（type, sector, data, status）
+- `process_queue()` メソッドで VirtQueue を処理
+- `read_sectors()` / `write_sectors()` メソッド
+- テスト用ディスクイメージの作成
+
+### 最終ゴール
+
+Phase 2 の最終ゴールは、Linux カーネルが VirtIO Block デバイスを認識し、ディスクイメージをマウントできるようにすることです。
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)